main/src/ca/uwaterloo/flix/api/lsp/LanguageServer.scala

```diff
@@ -316,7 +316,7 @@ class LanguageServer(port: Int) extends WebSocketServer(new InetSocketAddress("l
           // println(s"lsp/check: ${e / 1_000_000}ms")
 
           // Compute Code Quality hints.
-          val codeHints = CodeHinter.run(root, sources.keySet.toSet)(flix)
+          val codeHints = CodeHinter.run(root, sources.keySet.toSet)(flix, index)
```

Comment:
This line can be undone.

Reply:
Ok!

Reply:
I have tried to remove the index, but the compiler complains it cannot find it